### PR TITLE
Add Util tests

### PR DESCRIPTION
**Pull request by daiyiwhale**
👌

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @kennyt

File: src/js/utils/__tests__/Util-test.js [hunk 1]
```diff
@@ -4,6 +4,29 @@ var Util = require('../Util');
...
 
+  describe('#uniqueID', function () {
+
+    it('should return a unique ID each time it is called', function () {
+      let ids = Array(100).fill(null);
+      ids.forEach(function (value, index) {
+        ids[index] = Util.uniqueID();
+      });
+
+      let result = ids.reduce(function (allUnique, id, index) {
```
Comment (kennyt) [inline src/js/utils/__tests__/Util-test.js]: can we use `.every` here?

File: src/js/utils/__tests__/Util-test.js [hunk 2]
```diff
@@ -4,6 +4,29 @@ var Util = require('../Util');
...
+        return allUnique && !ids.includes(id, index + 1);
+      }, true);
+
+      expect(result).toBeTruthy();
+    });
+
+    it('should provide an integer', function () {
+      let id = Util.uniqueID();
+
+      expect(Number(id) === id && id % 1 === 0).toBeTruthy();
```
Comment (kennyt) [inline src/js/utils/__tests__/Util-test.js]: what's wrong with `typeof id ===  'number'`?

Comment (daiyiwhale) [inline src/js/utils/__tests__/Util-test.js]: because `typeof 3.14 === 'number'` is `true`

Comment (daiyiwhale) [inline src/js/utils/__tests__/Util-test.js]: oh I see what you're saying ahaha yes

Comment (kennyt) [inline src/js/utils/__tests__/Util-test.js]: but `Number(3.14) === 3.14` is also true

File: src/js/utils/__tests__/Util-test.js (hunk 1 above)
Comment (daiyiwhale) [inline src/js/utils/__tests__/Util-test.js]: oh yeah I forgot about that guy, fixed!

**Comment by kennyt** [comment]
merging on test pass

**Pull request merged**